modules/AlphaWalletFoundation/AlphaWalletFoundation/EtherClient/Models/ContractMethodCalls/Erc721TokenUriMethodCall.swift

```diff
@@ -8,11 +8,119 @@
 import Foundation
 import BigInt
 
+public struct Base64Decoder {
```

Comment:
Maybe move out to a separate file in AlphaWalletFoundation?

Reply:
yes, will do once done with it